device-types/Cisco/MDS-C9148T.yaml

```diff
@@ -0,0 +1,137 @@
+---
+manufacturer: Cisco
+model: MDS 9148T
+slug: ds-c9148t
```

Comment:
The slug should have `cisco-` prepended to it

Reply:
The slug has been prepended with "cisco-".